src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/integration/LicenceIntegrationTest.kt

```diff
@@ -635,6 +636,51 @@ class LicenceIntegrationTest : IntegrationTestBase() {
     assertThat(result?.prisonTelephone).isEqualTo("+44 276 54545")
   }
 
+  @Test
+  @Sql(
+    "classpath:test_data/clear-all-data.sql"
+  )
+  @Sql(
+    "classpath:test_data/seed-community-offender-manager.sql"
+  )
+  @Sql(
+    "classpath:test_data/seed-audit-events.sql"
+  )
+  @Sql(
```

Comment:
you can just have one annotation and an array of scripts

Reply:
yep